apps/vanilla/src/pages/blackjack/Classes/result-displayer.ts

```diff
@@ -0,0 +1,70 @@
+import { isHTMLElement } from '../guards/isHTMLElement';
+import { Subscriber } from '../types/subscriber';
+
+/** ResultData. */
+export class ResultData {
+	public constructor(public readonly diceResult: number[], public readonly totalScore: number) {}
+}
```

Comment:
```suggestion
export type Result {
	readonly diceResult: number[];
	readonly totalScore: number;
}
```

Let's use type for a data structure. Besides, the name of your data structure is not informative, it is also imported, so let's improve the naming